cli/cmd/root.go

```diff
@@ -222,6 +311,22 @@ func newCli() (*cli.CLI, error) {
 			return nil, fmt.Errorf("failed to create VMClarity API client: %w", err)
 		}
 
+		if exportCICDResults && scanResultID == "" {
+			i, err := cicdinitiator.NewInitiator(client, config, scanConfigID, scanConfigName, input, inputType)
```

Comment:
looks like `client, config, scanConfigID, scanConfigName, input, inputType` can be put in a `cicdinitiatorConfig`